app/test_split_integer.py

```diff
@@ -2,20 +2,26 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17, \
+        "The sum of the parts should be equal to the value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(25, 5) == [5, 5, 5, 5, 5], \
+        "The result length should be equal to the number of parts"
```

Comment:
We are not really testing for length - more appropriate would be to say that all elements in result should be equal to each other